net-irc/bitchx/bitchx-1.2.1.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="A text-based IRC client"
+DESCRIPTION="bitchx is an IRC client written in the C programming language.\
+It uses a text-mode user interface.
+Features:
+* Built-in ANSI color.
+* Ease of use. BitchX adds dozens of useful command aliases to reduce typing.
+* Built-in notify, protection, bot, and shit lists.
+* Built-in mass commands and tools.
+* Extended set of DCC commands and built-in CDCC offering.
+* Built-in link looker.
+* Extended scripting functionality, including functions found nowhere else.
+* Huge installed user base, an abundance of scripts, and a dedicated help \
+channel (#BitchX on EFNET).
+* Built-in screen client, allowing you to detach and re-attach to an irc session."
+HOMEPAGE="http://www.bitchx.com/"
```

Comment:
www.bitchx.org is the actual homepage, bitchx.com isn't associated with the project.

Reply:
Thanks,haven't noticed it.